addWecSimSource.m

```diff
@@ -4,4 +4,7 @@
 wecSimSource = fullfile(pwd,'source');
 addpath(genpath(wecSimSource));
 
-clear wecSimSource
\ No newline at end of file
+% Allow opening of Simulink models saved in a newer version
+set_param(0, 'ErrorIfLoadNewModel', 'off')
+
```

Comment:
This solution only works if users execute this script, which they may not do. For example, I have the source always added and never run this script. I think we should keep it in both places, and will make this revision.